admin/broadleaf-open-admin-platform/src/main/java/org/broadleafcommerce/openadmin/web/controller/entity/AdminBasicEntityController.java

```diff
@@ -417,24 +421,25 @@ public String addEntity(HttpServletRequest request, HttpServletResponse response
         return "ajaxredirect:" + getContextPath(request) + sectionKey + "/" + entity.getPMap().get("id").getValue();
     }
 
-    @RequestMapping(value = "/{id}/duplicate", method = RequestMethod.POST)
+    @RequestMapping(value = "/{id}/duplicate/{catalogId}", method = RequestMethod.POST)
```

Comment:
I think we need leave as it is and all "catalog" related changes should got to MT module.
for example there is MultiTenantAdminBasicEntityControllerExtensionHandler where we can adjust url of action to trigger something in MultiTenantAdminBasicEntityController(it overrides AdminBasicEntityController )